client/src/pages/Actions.ts

```diff
@@ -30,3 +88,41 @@ export const loginUser = (): IdentityToken | null => {
     });
   return identityToken;
 };
+
+export const getGeoLocation = (
+  successCallback: (position: GeoLocation) => void,
+  errorCallback?: (error: any) => void
+) => {
+  if (isWeb) {
+    // Use browser navigator
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(
+        successCallback,
+        errorCallback ? errorCallback : () => {}
+      );
+    }
+  } else {
+    // Use microapps API
+    microapps
+      .getCurrentLocation()
+      .then((loc: any) => Object.assign(emptyGeoLocation(), { coords: loc }))
+      .then((pos: GeoLocation) => successCallback(pos))
+      .catch((err: any) => (errorCallback ? errorCallback(err) : null));
+  }
+};
+
+export const getNearbyPlacesTest = (
```

Comment:
```suggestion
export const getNearbyPlaces = (
```

Reply:
Once again this is just a testing function for now, we will not actually be interacting directly with Places/Maps API from client-side in the future